app/assets/stylesheets/components.scss

```diff
@@ -34,6 +34,76 @@
     margin-top: 5px;
   }
 }
+// dgu meta message ============================================================
+.dgu-beta-message {
+  background: #0C5FA3;
+  margin-top: 40px;
+  margin-left: 15px;
+  margin-right: 15px;
+  color: white;
+  padding: 20px 20px 25px 20px;
+  a:link {
+    color: white;
+  }
+  a:visited {
+    color: white;
+  }
+  a:hover {
+    font-weight: bold;
+  }
+}
+
+// when beta message is displayed, spacing of adjacent elements must be adjusted____________
+
+// 1. Static pages and home page
+.dgu-beta-message__show {
```

Comment:
According to BEM naming conventions (http://getbem.com/naming/) the name should be
`.dgu-beta-message--show` as `show` is a modifier for the class